### PR TITLE
Changed contribution guidelines

### PR DESCRIPTION
**Pull request by Era-cell**
Hi, please review my PR😁

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -5,10 +5,11 @@ website about JSON Schema, [open a GitHub
 issue](https://github.com/jviotti/awesome-jsonschema/issues/new/choose) or
 add it directly to
 [`data.yaml`](https://github.com/jviotti/awesome-jsonschema/blob/master/data.yaml)
-and send a pull request. The README is auto-generated from `data.yaml`. You
-can locally render the README as follows:
+The README is auto-generated from `data.yaml`. You
+can render the README as follows:
```
Comment (jviotti) [inline CONTRIBUTING.md:9]: Line breaks seem weird here. We do 80 column hard wrapping. Can you re-format it?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -5,10 +5,11 @@ website about JSON Schema, [open a GitHub
...
 
 ```sh
 npm install
 npm start
 ```
+and then send a pull request.
```
Comment (jviotti) [inline CONTRIBUTING.md:15]: For consistency, please add a blank line in between the code block and the text. Also, this sentence should start with a capital letter, as it is a sentence of its own. 

Also, to be more direct, you can say something like:

```suggestion
After rendering the README with your changes, send a pull request that includes *both* the `data.yaml` and the `README.md`.
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (Era-cell) [reply CONTRIBUTING.md]: Yeah! that's a new thing. Thank you

**Review by jviotti — APPROVED** [review]
Looks great. Thanks!

**Pull request merged**